src/debug/utils.ts

```diff
@@ -76,10 +77,18 @@ export class PackageMap {
 					this.map[name] = uriToFilePath(rest);
 				else
 					this.map[name] = path.join(path.dirname(file), rest);
+
+				// If we map to "lib/" then this must be the local package so we can stash the name.
+				if (rest == "lib/")
```

Comment:
I think this works, but it is relying on an implementation detail (that could change).